cfpb/consumerfinance.gov
# Add prefooter border to RTL pages

Right-to-left arabic pages were missing the left-hand border on the prefooter, because it only added a right-hand border for LTR pages.


## Additions

- 


## Removals

-


## Changes

- Remove overridden `block` mixin and replace it with a margin-top.
- Add prefooter border to RTL pages.


## How to test this PR

1. Visit a LTR page with a prefooter and see that it has a border on the left, top, and right http://localhost:8000/about-us/budget-strategy/

2. Visit a RTL page with a prefooter and see that it has a border on the left, top, and right 0/language/ar/coronavirus/mortgage-and-housing-assistance/help-for-homeowners/protections-for-reverse-mortgage-borrowers/


## Screenshots

Before:
<img width="963" alt="Screen Shot 2023-06-06 at 5 54 45 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/04f8c78d-e3d3-4c6a-b11f-be7865f983c6">

After:
<img width="979" alt="Screen Shot 2023-06-06 at 5 54 27 PM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/ce802619-96b3-4d9d-aa41-63c3f1eeee33">
